packages/connect-ui/src/views/Error.tsx

```
@@ -181,6 +183,22 @@ const getTroubleshootingTips = (props: ErrorViewProps) => {
         });
     }
 
+    if (props.detail === 'connect-outdated') {
+        tips.push({
+            icon: 'question',
+            title: 'Deprecated version of Trezor Connect',
+            detail: {
+                steps: [
+                    <Step>
+                        {props.hostLabel} is using an outdated version of Trezor Connect. Please ask
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)